quickevent/app/quickevent/plugins/Runs/qml/reports/results_stageSpeaker.qml

```diff
@@ -128,8 +128,18 @@ Report {
 							Para {
 								width: 10
 								textFn: function() {
-									if(runnersDetail.rowData("disqualified"))
-										return qsTr("DISQ");
+									if(runnersDetail.rowData("disqualified")) {
```

Comment:
na zacatku je `import shared.qml.reports 1.0` tak si myslim, ze by tam mela jit pridat funkce `runStatusToString(row_data)`

Reply:
Tak to se mi uplne nedari. Vypada to, ze by se mi to mohlo podarit obdobne jako je ``OGTime`` pres ``libquickevent\libquickeventcore\js\ogtime.js``. Myslis ze bych to tak mohl zkusit udelat ?

Reply:
mrknu na to a napisu

Reply:
`libquickevent\libquickeventcore\js\ogtime.js` je ten spravny soubor, prikladam patch 
[0001-ogTimeToString-time_ms-function-added-to-ogtime.js.patch.gz](https://github.com/Quick-Event/quickbox/files/10366630/0001-ogTimeToString-time_ms-function-added-to-ogtime.js.patch.gz)

Reply:
Dekuju, s QML porad jeste bojuju.